src/main/java/com/dev/cinema/security/validation/ValidFieldRepeat.java

```diff
@@ -0,0 +1,29 @@
+package com.dev.cinema.security.validation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Constraint(validatedBy = PasswordValidator.class)
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ValidFieldRepeat {
+    String message() default "Password values don't match!";
+
+    String password();
+
+    String passwordRepeat();
+
+    Class<?>[] groups() default {};
+
+    Class<? extends Payload>[] payload() default {};
+
+    @Target({ ElementType.TYPE })
+    @Retention(RetentionPolicy.RUNTIME)
+    @interface List {
+        ValidFieldRepeat[] value();
+    }
```

Comment:
You don't need this block - it might use if you have several similar testing like - password-repeatPassword (eg + email - repeat-email, etc)

Reply:
Agree, thank you :)